python-client/giskard/scanner/llm/token_leakage.py

```diff
@@ -0,0 +1,132 @@
+import secrets
+from dataclasses import dataclass
+from typing import List, Optional, Sequence
+
+from langchain import PromptTemplate
+
+from ...datasets.base import Dataset
+from ...models.langchain import LangchainModel
+from ..decorators import detector
+from ..issues import Issue
+from ..logger import logger
+
+
+import pandas as pd
+
+import copy
+
+
+@detector("llm_token_leakage", tags=["text_generation", "leakage", "token_leakage"])
+class TokenLeakageDetector:
```

Comment:
Maybe you meant _prompt leakage_. Token leakage is not very clear.

Reply:
DONE